spartan/README.md

```diff
@@ -0,0 +1,109 @@
+# Spartan Kubernetes Cluster
+
+The Spartan Kubernetes cluster is used to run Aztec's public-facing node infrastructure. This README explains the Spartan cluster and its `staging` and `production` environments.
+
+# Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Kubernetes Primer](#kubernetes-primer)
+- [Aztec Staging and Production Environments](#aztec-staging-and-production-environments)
+  - [Helm and KIND Compatibility](#helm-and-kind-compatibility)
+- [Connecting to the Cluster](#connecting-to-the-cluster)
+  - [Connecting to Spartan using the CLI](#connecting-to-spartan-using-the-cli)
+  - [Connecting to Spartan using k8Lens](#connecting-to-spartan-using-k8lens)
+- [Auxiliary Services](#auxiliary-services)
+  - [Prometheus and Grafana](#prometheus-and-grafana)
+
+## Prerequisites
+
+Before connecting to the Spartan cluster, you will need to complete the following items:
+
+- [Install AWS CLI Tools](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
```

Comment:
These instructions are very workable, though I'll just note for me the gold standard is a script akin to our bootstrap.sh script that automates as much as possible

Reply:
Here AWS is fine, but perhaps a callout to spartan/scripts/setup_local_k8s.sh

Reply:
ekctl -> eksctl?

Reply:
(not intending this to be a blocker for this PR in any way)